### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by axic** [comment]
> The `evmc_set_option()` not returns more information about the failure cause.

not -> now

**Comment by chfast** [comment]
Anything else?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,32 @@
 
 ## [6.0.0-dev] - unreleased
```
Comment (axic) [inline CHANGELOG.md]: Not sure when is this updated?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,32 @@
...
 
-- Improved: [[#119](https://github.com/ethereum/evmc/pull/119)]
+- Added: [[#116](https://github.com/ethereum/evmc/pull/116)]
+  [EVMC Host implementation example](https://github.com/ethereum/evmc/blob/master/examples/example_host.cpp).
+- Added: [[#127](https://github.com/ethereum/evmc/pull/127)]
+  Support for Constantinople SSTORE net gas metering.
+- Added: [[#133]](https://github.com/ethereum/evmc/pull/133)
+  Support for Constantinople CREATE2 salt in Go bindings.
+- Added: [[#144](https://github.com/ethereum/evmc/pull/144)]
+  A VM can now report its **capabilities** (i.e. EVM and/or eWASM).
```
Comment (axic) [inline CHANGELOG.md]: We were using `ewasm` since January.

File: CHANGELOG.md (hunk 1 above)
Comment (chfast) [reply CHANGELOG.md]: Should be update with a new release.

**Pull request merged**